README.md

```diff
@@ -1,17 +1,59 @@
 # Getting Started with the Vue Date Range Picker
 
-A quick-start project that shows how to work with the Syncfusion Date Range Picker in a Vue application. This project contains simple code customization, as well as some important features like setting specific date range values, range restriction, day spans, formats, and a month-range picker.
-
-Refer to the following documentation for the Syncfusion Vue Date Range Picker component: 
-https://ej2.syncfusion.com/vue/documentation/daterangepicker/getting-started-vue3/
-
-Check out this online example of the Syncfusion Vue Date Range Picker component: 
-https://ej2.syncfusion.com/vue/demos/#/material/daterangepicker/default.html
+The [Vue Date Range Picker](https://www.syncfusion.com/vue-components/vue-date-range-picker?utm_source=github&utm_medium=listing&utm_campaign=vue-date-range-picker-github-samples) is a lightweight and mobile-friendly component that allows end users to select start and end date values as a range from a calendar pop-up or by entering values directly in the HTML input text box.A quick-start project that shows how to work with the Syncfusion Date Range Picker in a Vue application. This project contains simple code customization, as well as some important features like setting specific date range values, range restriction, day spans, formats, and a month-range picker.
 
```

Comment:
Space needed here.
![image](https://github.com/SyncfusionExamples/getting-started-with-the-vue-date-range-picker/assets/95272441/dfc63b10-c0bd-409e-98d1-72d77702f02f)

Reply:
Done